File: src/WebUI/ClientApp/src/domain/models/Task.ts
import { Category } from "./Category";

export type Task = {
  id: number;
  listId: number;
  position: number;
  name: string;
  businessValue: string;
  description: string;
  done: boolean;
  categories: Category[];
  lastModifiedBy?: string;
  lastModified?: string;
};

File: src/Application/Entities/Task.cs
using Chrono.Application.Entities.Common;

namespace Chrono.Application.Entities;

public class Task : BaseAuditableEntity
{
    public int ListId { get; set; }
    public int Position { get; set; }
    public string Name { get; set; }
    public string BusinessValue { get; set; }
    public string Description { get; set; }
    public bool Done { get; set; }
    public IList<TaskCategory> Categories { get; } = new List<TaskCategory>();
    public TaskList List { get; set; }
}
